tools/blobstorage-backupdata/JObjectExtensions.cs

```diff
@@ -24,4 +25,39 @@ internal static DateTime GetDateTime(this JObject jObject)
     {
         return DateTime.Parse(jObject["_meta"]?["updated"]?.ToString()!);
     }
+
+    /// <summary>
+    /// Gets the blob definition url.
+    /// </summary>
+    /// <param name="jObject">Json object extension param.</param>
+    /// <returns>returns the constructed definition url from the coordinates field</returns>
+    internal static string ConstructBlobUrl(this JObject jObject)
+    {
+        if (jObject?["coordinates"]?.Type == null || jObject["coordinates"]!.Type == JTokenType.Null)
```

Comment:
When we query from the db, coordinates were excluded in the project on Line 66.  So without modifying the find option, coordinates will not be available.  There is a _id field from the db, e.g.  "_id": "pypi/npmjs/-/lodash/3.4.0", which is similar to what is needed here.  The _id is constructed by `${type}/${provider}/${namespace ?? '-'}/${name}/${revision}`.toLowerCase()